pages/new-aduan-warga/detail/index.vue

```diff
@@ -73,20 +73,20 @@ export default {
     },
   },
   mounted() {
-    this.fetchComplaintData()
+    this.getDetailComplaint()
+    this.getTrackingComplaint()
   },
   methods: {
     formatDate,
     getSelected(value) {
       this.selectedTab = value
     },
-    async fetchComplaintData() {
+    async getDetailComplaint() {
       this.isLoading = true
       try {
-        const response = await this.$axios.get(
-          '/v1/aduan/complaints/:id/status',
-          { params: { search: this.search } }
-        )
+        const response = await this.$axios.get('/v1/aduan/complaints/:id', {
+          params: { search: this.search },
+        })
```

Comment:
is this URL correct @rayfajars ? usually `:id` mean a dynamic id

Reply:
oh lupa diganti bli, ini tuh API nya belum ready jadi nangung mau diubah lagi

saya kasih todo dulu ya bli

fix [here](https://github.com/jabardigitalservice/super-app-web/pull/111/commits/9751ae136a086f52c7bef61cef84e05d2f0e2c62)